skyportal/handlers/api/galaxy.py

```diff
@@ -115,26 +115,26 @@ def get(self, catalog_name=None):
               schema:
                 type: string
               description: |
-                Event time in isot format
+                Event time in ISO 8601 format (`YYYY-MM-DDTHH:MM:SS.sss`).
             - in: query
               name: localizationName
               schema:
                 type: string
               description: |
-                Name of localization / skymap to use
+                Name of localization / skymap to use. Can be found in Localization.localization_name
```

Comment:
What does `Localization.localization_name` mean to users?  I guess what I'm asking with the docstrings is to use text that makes it obvious for newcomers to the API on what to do to obtain this value, or how to use it.

Reply:
Ah yes. I added more information (the endpoint one could use or the fact that it is in the table on the GcnEvents page).